typed_ast/ast35.py

```diff
@@ -12,10 +12,11 @@
 
     In particular, the `FunctionDef`, `Assign`, `For`, and `With` classes all
     have a `type_comment` field which contains a `str` with the text of the
-    type comment.  `parse` has been augmented so it can parse function
-    signature types when called with `mode=func_type`.  Finally, `Module` has
-    a `type_ignores` field which contains a list of lines which have been
-    `# type: ignored`.
+    type comment.  Per-argument function comments are put into the annotation
+    field of each argument.  `parse` has been augmented so it can parse
```

Comment:
This paragraph is probably best turned into bullets at this point -- the four sentences are fairly independent of each other, and the transition from "per-argument function comments" to "`parse` has been augmented" is particularly abrupt.